Utilies/CustomLogger.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+
+class LogGen:
+    @staticmethod
+    def loggen():
+        logging.basicConfig (filename=".\\Logs\\automation.logs",
+                             format='%(asctime)s: %(levelname)s: %(message)s',
+                            datefmt='%m/%d/%Y %I:%M:%S %p')
+
+                            
+        logger = logging.getLogger()
+        logger.setLevel(logging.INFO)
+        return logger
+C:\Users\WELCOME/.ssh/id_ed25519.pub
```

Comment:
What is this last line 15 ? If this is a typo, please remove and push again

Reply:
Thanks, removed the typo.